src/Microsoft.AspNetCore.Mvc.Razor/RazorView.cs

```diff
@@ -233,10 +240,25 @@ private Task RenderPageCoreAsync(IRazorPage page, ViewContext context)
 
             if (bodyWriter.IsBuffering)
             {
-                // Only copy buffered content to the Output if we're currently buffering.
-                using (var writer = _bufferScope.CreateWriter(context.Writer))
+                // If IsBuffering - then we've got a bunch of content in the view buffer. How to best deal with it
```

Comment:
Think I'm missing something though it's probably an existing facet of the design. The layout page was passed `bodyWriter.Buffer` and may have rendered that content before we get here. What prevents writing that information out twice?